tasks/retrieval/msrvtt/msrvtt_process.py

```diff
@@ -0,0 +1,58 @@
+import json
+import os
+import os.path as osp
+from PIL import Image
+import subprocess
+import zipfile
+import os
+# from dataloaders.rawframe_util import RawFrameExtractor
+def process(cfg):
+    download_and_extract_dataset(cfg.dataset_path,"./",cfg.extract_dir)
+    
+ 
+def download_and_extract_dataset(repo_id, cache_dir, extract_dir):
+    """
+    Download Hugging Face dataset and extract it to the specified path.
+
+    Parameters:
+        repo_id (str): The name of the dataset (e.g., "shiyili1111/MSR-VTT").
+        cache_dir (str): The directory to cache the downloaded file.
+        extract_dir (str): The directory to extract the dataset.
+    """
+    # Ensure the cache directory exists

```

Comment:
`cache_dir = osp.join(FLAGEVALMM_DATASETS_CACHE_DIR, processed_dataset_path, split)`
and use the same `cache_dir` and `extract_dir`